execution_engine/src/core/runtime_context/mod.rs

```diff
@@ -767,6 +771,7 @@ where
                 false
             }
             Key::SystemContractRegistry => false,
+            Key::EraSummary => true,
```

Comment:
is there a cltyped for it?

Reply:
ok, i see it...era_info already had it:
```
impl CLTyped for EraInfo {
    fn cl_type() -> CLType {
        CLType::List(Box::new(SeigniorageAllocation::cl_type()))
    }
}
```

( SeigniorageAllocation::cl_type() == Any )